sdk/servicebus/service-bus/test/internal/node/packageInfo.spec.ts

```diff
@@ -11,7 +11,10 @@ import { packageJsonInfo } from "../../../src/util/constants";
 // following test is in place to ensure the values in package.json and in this file are consistent
 describe("Ensure package name and version are consistent in SDK and package.json", function(): void {
   it("Ensure constants.ts file is consistent with package.json", () => {
-    const packageJsonFilePath = path.join(__dirname, "../../../../package.json");
+    const packageJsonFilePath = ["../../../../package.json", "../../../package.json"]
+      .map((pj) => path.join(__dirname, pj))
+      .filter((pj) => fs.existsSync(pj))[0];
```

Comment:
Reason for this change?

Reply:
If you swap to mocha -r tsnode/register (ie, don't compile, just run with ts-node) then the path depth changes.

So if you're using both that and the dist-esm version for tests you have to handle the different path depths.

Reply:
Can you add a comment for that? We may have similar tests in other packages, can you log an issue to do a review?

Reply:
Added!